### PR TITLE
Add FDI base info

### PR DESCRIPTION
**Pull request by Lennonka**
We have a request to add this info.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 1]
```diff
@@ -37,17 +37,23 @@ endif::[]
...
+[NOTE]
+====
+ifdef::satellite[]
+The Foreman Discovery image provided with {Project} is based on {EL} 8.
+endif::[]
+ifndef::satellite[]
+{Project} provides multiple versions of the Foreman Discovery image (FDI):
+
+* FDI version 4.0 and newer are based on {EL} 8
+* FDI versions older than 4.0 are based on {EL} 7
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:49]: Should I be more specific and say [CentOS Stream](https://github.com/theforeman/foreman-discovery-image/blob/master/00-repos-centos8.ks)?

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 2]
```diff
@@ -37,17 +37,23 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
+[NOTE]
+====
+ifdef::satellite[]
+The Foreman Discovery image provided with {Project} is based on {EL} 8.
+endif::[]
+ifndef::satellite[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:45]:
```suggestion
ifdef::satellite[]
The Foreman Discovery image provided with {Project} is based on {EL} 8.
endif::[]
ifdef::satellite[]
The Foreman Discovery image provided with {Project} is based on CentOS Stream 8.
endif::[]
ifndef::satellite,orcharhino[]
```

Can you please verify & ACK this @jloeser?

Comment (Lennonka) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:45]: You probably mean orcharhino for the second `ifdef` :)

Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:45]: Yes :see_no_evil:

Comment (jloeser) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:45]: If we ship official FDI from Foreman then it's also CentOS Stream 8.

**Comment by jloeser** [comment]
LGTM.

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- b0646249ea..b91ff53a29  3.10 -> 3.10
- d255fd4b25..6237e738e9  3.9 -> 3.9
- 0671795d27..43ca830c89  3.8 -> 3.8
- ba4714c3a7..dc38fcdd33  3.7 -> 3.7
- e5e022cffb..86b9773384  3.6 -> 3.6
- 1d44411b56..dabd00b5bd  3.5 -> 3.5

**Pull request merged**